lib/features/onboarding/ui/widgets/expandable_search.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:flutter/material.dart';
+import 'package:miny_design_system/miny_design_system.dart';
+
+import '../../utilities/onboarding_constants.dart';
+
+class ExpandableSearchField extends StatefulWidget {
+  final List<String> allItems;
+  final String? selected;
+  final ValueChanged<String> onSelected;
+
+  const ExpandableSearchField({
+    super.key,
+    required this.allItems,
+    required this.onSelected,
```

Comment:
if null, should not required
```suggestion
    this.onSelected,
```

Reply:
But onSelected is not null